toml/src/main/kotlin/org/rust/toml/completion/LocalCargoTomlDependencyCompletionProvider.kt

```diff
@@ -65,6 +68,8 @@ class LocalCargoTomlDependencyCompletionProvider : TomlKeyValueCompletionProvide
                 index.toDouble()
             )
         }
-        result.addAllElements(elements)
+        result.withRelevanceSorter(CompletionSorter.emptySorter().weigh(object : LookupElementWeigher("priorityWeigher", true, false) {
```

Comment:
WDYT about using "priority" instead of "priorityWeigher"? What's the reason for using "priorityWeigher"?